lib/encumber.rb

```diff
@@ -374,42 +377,74 @@ def move_mouse_to_point(x, y)
 
     end
 
-    def simulate_left_click(xpath, flags=[])
+    def location_for_simulate_mouse_event(original_location, current_location, destination_location, i)
```

Comment:
Yes, I moved the mouse simulation in `move_mouse_to_point` as it was.
`location_for_simulate_mouse_event` only compute x and y now.